packages/search-ui/src/SearchDriver.ts

```diff
@@ -163,6 +163,12 @@ class SearchDriver {
       },
       {}
     ) as actions.SearchDriverActions;
+
+    this.actions = {
```

Comment:
I feel this might need more thought. With this change, we are allowing some connectors to support more functionality than others which can be a little gotcha here with some components depending on particular connectors being used. I also think `logout` would not exist within the types here as its not part of the SearchDriverAction types.